app/main.py

```diff
@@ -11,3 +11,10 @@
 #     print(Actor.objects.all())
 #     Actor.objects.delete(1)
 #     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+    Actor.objects.delete(1)
```

Comment:
You are deleting an object with an id of 1 here, but you have not created any objects before this. This will likely result in an error.